src/tipowof10.c

```diff
@@ -167,7 +167,7 @@ const vui128_t vtifrexpof10[] =
   (vui128_t) (__int128 ) 4768371582031250ll, /* 10**22 */
 
 #ifdef __clang__
-// Clang does support constant folding for __int128
+// Clang does not support constant folding for __int128
```

Comment:
should this be handled in 'configure' ?

Reply:
I will think about this for the next round.